include/solver_pic.h

```diff
@@ -388,113 +404,114 @@ struct PIC_State {
     extra_container_type marker_extras;
     const std::vector<value_type> quasi_neutrality_coef;
     field_type field;
+    bool drift_center_transformation_switch;
```

Comment:
It seem like this parameter is already a member of `Parameter` class, thus we dont need to keep another copy here.